Creating release for self-hosted GitHub Enterprise fails

our environment is self-hosted GitHub Enterprise and self-hosted runners.

when i try to create a release, it fails with the following error:

```
none of the git remotes configured for this repository point to a known GitHub host. To tell gh about a new GitHub host, please use `gh auth login`
Error: Process completed with exit code 1.
```

here's the workflow code:

```yaml
      # create release if create_release is true.
      - name: Create a Release
        if: ${{ github.event.inputs.create_release }}
        uses: elgohr/Github-Release-Action@v5
        env:
          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          title: ${{ github.event.inputs.message }}
          tag: ${{ env.TAG }}
```

i've tried adding a step prior to this that performs a `gh auth`, but i'm not certain that persists thru steps.

other action steps succeed, such as a `git push` command when the new tag is pushed up to our GitHub, using the `${{ secrets.GITHUB_TOKEN }}`.

does this action work within the checked out repo code like other action steps do? could this be an issue with `workdir`?

or am i missing something else?

--- a/README.md
+++ b/README.md
@@ -54,3 +54,24 @@ permissions:
 ```
 
 to the concrete job creating the release. For more details see the [documentation on token permissions.](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#modifying-the-permissions-for-the-github_token)
+
+### Use with GitHub Enterprise
+
+To publish your release to self-hosted GitHub Enterprise, include the following environment variables:
+
+GH_ENTERPRISE_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+GH_HOST: yourgithub.company.com
+
+For example:
+
+```yaml
+      - name: Create Release
+        if: ${{ github.event.inputs.create_release }}
+        uses: elgohr/Github-Release-Action@v5
+        env:
+          GH_ENTERPRISE_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_HOST: yourgithub.company.com
+        with:
+          title: "New release"
+          tag: "v1.0.1"
+```

--- a/README.md
+++ b/README.md
@@ -57,11 +57,7 @@ to the concrete job creating the release. For more details see the [documentatio
 
 ### Use with GitHub Enterprise
 
-To publish your release to self-hosted GitHub Enterprise, include the following environment variables:
-
-GH_ENTERPRISE_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-GH_HOST: yourgithub.company.com
-
+To publish your release to self-hosted GitHub Enterprise, include `GH_ENTERPRISE_TOKEN` and `GH_HOST` as environment variables.  
 For example:
 
 ```yaml

